codemod/src/scripts/modify.ts

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
+
```

Comment:
I will write docs for these in the documentation phase (small readme about how to use this runner and plugins)